Appraisals

```diff
@@ -24,3 +24,7 @@ end
 appraise 'rails8_0' do
   gem 'rails', '~> 8.0.0'
 end
+
+appraise 'rails8_1' do
+  gem 'rails', '~> 8.1.0'
```

Comment:
There’s a `< 8.1` in `activerecord_views.gemspec` which needs to change to `< 8.2` for `bundle install --gemfile gemfiles/rails8_1.gemfile` to succeed.